ivy_tests/test_ivy/helpers/multiprocessing.py

```diff
@@ -0,0 +1,327 @@
+# global
+import sys
+import importlib
+
+# local
+from .testing_helpers import (
+    _get_supported_devices_dtypes_helper,
+    _get_method_supported_devices_dtypes_helper,
+)
+from .function_testing import (
+    test_function_backend_computation,
+    test_function_ground_truth_computation,
+    test_method_backend_computation,
+    test_method_ground_truth_computation,
+    test_gradient_backend_computation,
+    test_gradient_ground_truth_computation,
+)
+
+
+def backend_proc(input_queue, output_queue):
+    # first argument is going to be the framework and its path
+    framework = input_queue.get()
+    path = "/opt/fw/" + framework
+    sys.path.insert(1, path)
+    framework = framework.split("/")[0]
+    framework = importlib.import_module(framework)
+    # if jax, do more stuff
+    if framework.__name__ == "jax":
+        framework.config.update("jax_enable_x64", True)
+
+    while True:
+        # subsequent arguments will be passed
+        data = input_queue.get()
```

Comment:
Can we use a dict instead of a list?

Reply:
You mean the data stream ? It needs to be a list or a tuple with indexing but I guess we can use a dictionary too, will need more refactoring of the code.